semeio/workflows/misfit_preprocessor/config.py

```diff
@@ -221,17 +221,28 @@ def _inject_default_t(config):
 AUTO_SCALE_SCHEMA = {
     cs.MetaKeys.Type: cs.types.NamedDict,
     cs.MetaKeys.Description: (
-        "The {sc} clustering is supporting multiple parameters."
-    ).format(sc=AUTO_SCALE),
+        f"The {AUTO_SCALE} clustering is supporting multiple parameters. The "
+        "the job will first run pca analysis on all the data to determine the number "
+        "of components. Then spearman correlation is performed on the simulated data"
+        " to create a correlation matrix. Clustering is then performed on the "
+        "correlation matrix using the input from the first pca analysis, where the "
+        "number of clusters equals the number of components. Then pca is performed "
+        "on the individual clusters. Observations in each cluster is then scaled with "
```

Comment:
```suggestion
        "on the individual clusters. Observations in each cluster are then scaled with "
```
Subject is observations?